### PR TITLE
Add instrumented service.

### PR DESCRIPTION
**Pull request by Random-Liu**
Move all grpc handler logs into a instrumented service, so that we don't need to care about it in the real service.

Signed-off-by: Lantao Liu <lantaol@google.com>

### CODE REVIEW

**Comment by abhinandanpb** [comment]
Should we convert those logs to something like debugf in logrus. I see that glog has severity level may be print those logs only if the logging level is set appropriately ? Instead of an instrumented code.

**Comment by Random-Liu** [comment]
@abhinandanpb I did this not for the log level. :) Just extract the log logic into an outside wrapper, so that we don't need to see this everywhere. :)

File: pkg/server/instrumented_service.go [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+
+func newInstrumentedService(c *criContainerdService) CRIContainerdService {
+	return &instrumentedService{criContainerdService: c}
+}
+
+func (in *instrumentedService) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandboxRequest) (res *runtime.RunPodSandboxResponse, err error) {
+	glog.V(2).Infof("RunPodSandbox with config %+v", r.GetConfig())
+	defer func() {
+		if err != nil {
+			glog.Errorf("RunPodSandbox for %+v returns error %v", r.GetConfig().GetMetadata(), err)
```
Comment (abhinandanpb) [inline pkg/server/instrumented_service.go]: ```glog.Errorf("RunPodSandbox for %+v failed, err: %v", r.GetConfig().GetMetadata(), err)```  
would be better ?

Comment (Random-Liu) [inline pkg/server/instrumented_service.go]: Will update all.

Comment (Random-Liu) [inline pkg/server/instrumented_service.go]: Done.

**Comment by abhinandanpb** [comment]
LGTM

**Comment by Random-Liu** [comment]
This touches many files, and relatively lower priority. I'll hold this until https://github.com/kubernetes-incubator/cri-containerd/pull/138 and https://github.com/kubernetes-incubator/cri-containerd/pull/128 get merged.

/cc @abhinandanpb

**Pull request merged**